### ISSUE TITLE
rspec upload errors in logs

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-portal#781]
```
[Wed Aug 28 16:56:47 2013] [error] [client 152.2.131.202] PHP Warning:  DOMDocument::loadXML(): xmlParseEntityRef: no name in Entity, line: 27 in /var/www/secure/rspecupload.php on line 49, referer: https://portal.geni.net/secure/rspecupload.php
[Wed Aug 28 16:56:47 2013] [error] [client 152.2.131.202] PHP Warning:  DOMDocument::loadXML(): xmlParseEntityRef: no name in Entity, line: 27 in /var/www/secure/rspecupload.php on line 49, referer: https://portal.geni.net/secure/rspecupload.php
[Wed Aug 28 16:56:47 2013] [error] [client 152.2.131.202] PHP Warning:  DOMDocument::loadXML(): xmlParseEntityRef: no name in Entity, line: 50 in /var/www/secure/rspecupload.php on line 49, referer: https://portal.geni.net/secure/rspecupload.php
[Wed Aug 28 16:56:47 2013] [error] [client 152.2.131.202] PHP Warning:  DOMDocument::loadXML(): xmlParseEntityRef: no name in Entity, line: 50 in /var/www/secure/rspecupload.php on line 49, referer: https://portal.geni.net/secure/rspecupload.php
[Wed Aug 28 16:56:47 2013] [error] [client 152.2.131.202] PHP Notice:  Trying to get property of non-object in /var/www/secure/rspecupload.php on line 56, referer: https://portal.geni.net/secure/rspecupload.php
[Wed Aug 28 16:56:47 2013] [error] [client 152.2.131.202] PHP Warning:  Invalid argument supplied for foreach() in /var/www/secure/rspecupload.php on line 56, referer: https://portal.geni.net/secure/rspecupload.php
```

 _Imported from trac ticket [#781](http://trac.gpolab.bbn.com/proto-ch/ticket/781),  created by ahelsing on 08-30-2013 at 17:22, last modified: 09-05-2013 at 17:05_

**Comment by tcmitchell** [comment]
See issue #720. I anticipate that this issue is fixed in milestone:2.1.

_Trac comment by tmitchel (github user: tcmitchell) on 09-03-2013 at 09:55_

**Comment by tcmitchell** [comment]
Resolving because of the similarity to the errors in issue #720. Without the actual RSpec that was uploaded this one's awfully hard to test.

_Trac comment by tmitchel (github user: tcmitchell) on 09-05-2013 at 17:05_
